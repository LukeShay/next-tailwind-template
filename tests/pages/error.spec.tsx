import { create } from 'react-test-renderer';
import React from 'react';
import Error from '../../pages/error';

describe('error', () => {
  it('should match snapshot', () => {
    expect.hasAssertions();
    const page = create(<Error />);
    expect(page.toJSON()).toMatchSnapshot();
  });
});
