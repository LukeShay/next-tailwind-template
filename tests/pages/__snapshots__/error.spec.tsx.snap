// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error should match snapshot 1`] = `
<h1>
  Uh oh, You went somewhere that doesn't exist!
</h1>
`;
